features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
```

Comment:
``In order to buy goods with correct taxes applied``?

Feature benefit cannot be feature itself...